src/content/app-platform/getting-started/index.md

```diff
@@ -88,9 +88,9 @@ CRs.
 ```nohighlight
 kubectl gs get appcatalogs
 
-NAME                    CATALOG URL                                                   CREATED
-giantswarm              https://giantswarm.github.io/giantswarm-catalog/              17 Jul 20 17:28 CEST
-giantswarm-playground   https://giantswarm.github.io/giantswarm-playground-catalog/   17 Jul 20 17:28 CEST
+NAME                    CATALOG URL
+giantswarm              https://giantswarm.github.io/giantswarm-catalog/
+giantswarm-playground   https://giantswarm.github.io/giantswarm-playground-catalog/
```

Comment:
These dates are causing wrapping.I don't think they are needed to understand what is going on. So I've just removed them.